rpc/src/rpc.rs

```diff
@@ -3373,15 +3393,35 @@ pub mod utils {
                 let pre_execution_accounts = if let Some(pre_tx_accounts) =
                     bundle_output.pre_tx_execution_accounts().get(index)
                 {
-                    try_encode_accounts(pre_tx_accounts, account_encoding)?
+                    try_encode_accounts(
+                        pre_tx_accounts,
+                        account_encoding,
+                        bank,
+                        Some(&account_overrides),
```

Comment:
do you need to use account overrides here? you should have access to the accounts in the execution results

Reply:
It comes from the post accounts, but needs to be converted to the shape `get_encoded_account` expects, which is exactly your `AccountOverrides` type. Upstream doesn't have that type, but it's the same shape.

Are you suggesting writing a version of `get_encoded_account` that takes the Jito `post_accounts: Vec<(Pubkey, AccountSharedData)>` directly? I thought it'd be a smaller change to re-use `get_encoded_account` from upstream by reshaping the `post_accounts` to match it.

I think I'd also need to write upsert logic to combine the outputs of the transactions in the bundle since we encode the accounts after each transaction and not just at the end. I thought it was better to re-use `AccountOverrides::upsert_account_overrides` instead of writing custom logic for that.